schema/validator.go

```diff
@@ -54,6 +55,28 @@ func (v Validator) Validate(src io.Reader) error {
 			"schema %s: unable to validate", v)
 	}
 
+	if v == MediaTypeManifest {
```

Comment:
@wking `validate --type manifest` type seems not to jump into `manifest.validate`, instead into `Validator.Validate` directly. So, `manifest.validate` can't be effective.